src/components/settingsMenu.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Box,
+  Tab,
+  Tabs,
+  AppBar,
+  Typography,
+} from '@material-ui/core'
+
+import GurbaniSwitchToggle from './editors/gurbaniToggles'
+import LayoutEditor from './editors/layoutEditor'
+import BackgroundEditor from './editors/backgroundEditor'
+import FontEditor from './editors/fontEditor'
+import VishraamEditor from './editors/vishraamEditor'
+import ExportEditor from './editors/exportEditor'
+
+// WIP: need to organize code
+
+const TabPanel = props => {
+  const {
+    children, value, index, ...other
```

Comment:
Keep the braces and all values on one line, OR one value per line and braces on their own line (if the former makes it too long)